The new package has not been pushed to GH during the release

That's the state after runnign the release script:

![image](https://user-images.githubusercontent.com/156149/53565339-7e622480-3b59-11e9-8747-c57337d300fa.png)

Also: 

![image](https://user-images.githubusercontent.com/156149/53565412-b701fe00-3b59-11e9-8b03-5bcc1a106af4.png)

It will also fail for publishing on NPM:

```
Checking "@ckeditor/ckeditor5-mention"...
⚠️   Execute: "npm show @ckeditor/ckeditor5-mention version" in "/Users/pomek/Projects/ckeditor/ckeditor5/packages/ckeditor5-mention".
npm ERR! code E404
npm ERR! 404 '@ckeditor/ckeditor5-mention' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! 404
npm ERR! 404  '@ckeditor/ckeditor5-mention@latest' is not in the npm registry.
npm ERR! 404 Your package name is not valid, because
npm ERR! 404  1. name can only contain URL-friendly characters
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pomek/.npm/_logs/2019-03-14T10_43_08_710Z-debug.log
```

--- a/packages/ckeditor5-dev-env/lib/release-tools/tasks/bumpversions.js
+++ b/packages/ckeditor5-dev-env/lib/release-tools/tasks/bumpversions.js
@@ -68,7 +68,7 @@ module.exports = function bumpVersions( options ) {
 		.then( () => {
 			process.chdir( cwd );
 
-			logProcess( `Finished updating versions of ${ chalk.underline( pathsCollection.matched.size ) } packages.` );
+			logProcess( `Finished updating versions of ${ chalk.underline( pathsCollection.matched.size ) } package(s).` );
 			logDryRun( 'Because of the DRY RUN mode, nothing has been changed. All changes were reverted.' );
 		} )
 		.catch( err => {

--- a/packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js
+++ b/packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js
@@ -143,7 +143,7 @@ module.exports = function releaseSubRepositories( options ) {
 		.then( () => {
 			process.chdir( cwd );
 
-			logProcess( `Finished releasing ${ chalk.underline( releasedPackages.size ) } packages.` );
+			logProcess( `Finished releasing ${ chalk.underline( releasedPackages.size ) } package(s).` );
 			logDryRun( 'Because of the DRY RUN mode, nothing has been changed. All changes were reverted.' );
 		} )
 		.catch( err => {
@@ -239,9 +239,9 @@ module.exports = function releaseSubRepositories( options ) {
 
 			log.info( `\nChecking "${ chalk.underline( packageJson.name ) }"...` );
 
-			const npmVersion = exec( `npm show ${ packageJson.name } version` ).trim();
+			const npmVersion = getVersionFromNpm( packageJson.name );
 
-			logDryRun( `Versions: package.json: "${ releaseDetails.version }", npm: "${ npmVersion }".` );
+			logDryRun( `Versions: package.json: "${ releaseDetails.version }", npm: "${ npmVersion || 'initial release' }".` );
 
 			releaseDetails.npmVersion = npmVersion;
 			releaseDetails.shouldReleaseOnNpm = npmVersion !== releaseDetails.version;
@@ -256,6 +256,26 @@ module.exports = function releaseSubRepositories( options ) {
 
 			return Promise.resolve();
 		} );
+
+		// Checks whether specified `packageName` has been published on npm.
+		// If so, returns its version. Otherwise returns `null` which means that
+		// this package will be published for the first time.
+		function getVersionFromNpm( packageName ) {
+			try {
+				return exec( `npm show ${ packageName } version` ).trim();
+			} catch ( err ) {
+				const errorAsArray = err.message.split( '\n' ).slice( 0, 2 );
+
+				if (
+					errorAsArray[ 0 ].endsWith( 'npm ERR! code E404' ) &&
+					errorAsArray[ 1 ] === `npm ERR! 404 '${ packageName }' is not in the npm registry.`
+				) {
+					return null;
+				}
+
+				throw new Error( err );
+			}
+		}
 	}
 
 	// Checks for which packages GitHub release should be created. It compares version defined in `package.json`
@@ -316,7 +336,7 @@ module.exports = function releaseSubRepositories( options ) {
 				github.repos.getLatestRelease( requestParams, ( err, responses ) => {
 					if ( err ) {
 						// No releases on GitHub. It will be the first one.
-						if ( err.message.toLowerCase() == 'not found' ) {
+						if ( err.status.toLowerCase() == 'not found' ) {
 							return resolve( {
 								data: {
 									tag_name: null

--- a/packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js
+++ b/packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js
@@ -247,7 +247,7 @@ module.exports = function releaseSubRepositories( options ) {
 			releaseDetails.shouldReleaseOnNpm = npmVersion !== releaseDetails.version;
 
 			if ( releaseDetails.shouldReleaseOnNpm ) {
-				logDryRun( 'Package will be released.' );
+				log.info( '✅  Added to release.' );
 
 				releasesOnNpm.add( repositoryPath );
 			} else {
@@ -314,11 +314,11 @@ module.exports = function releaseSubRepositories( options ) {
 					releaseDetails.shouldReleaseOnGithub = githubVersion !== releaseDetails.version;
 
 					if ( releaseDetails.shouldReleaseOnGithub ) {
-						logDryRun( 'Package will be published.' );
+						log.info( '✅  Added to release.' );
 
 						releasesOnGithub.add( repositoryPath );
 					} else {
-						log.info( '❌  Nothing to publish.' );
+						log.info( '❌  Nothing to release.' );
 					}
 				} )
 				.catch( err => {

--- a/packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js
+++ b/packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js
@@ -273,7 +273,7 @@ module.exports = function releaseSubRepositories( options ) {
 					return null;
 				}
 
-				throw new Error( err );
+				throw err;
 			}
 		}
 	}

